home/tests/test_content_import_export.py

```diff
@@ -188,6 +195,22 @@ def _normalise_button_pks(body: DbDict, min_pk: int) -> DbDict:
                 button = button | {"value": v | {"page": v["page"] - min_pk}}
             buttons.append(button)
         value = value | {"buttons": buttons}
+    print("nomralised bttons ", body | {"value": value})
```

Comment:
Can probably remove this :-)

Reply:
Haha, oh dear, my typo-ridden debug print forever immortalized in git history